lib/ui/platform_dispatcher.dart

```diff
@@ -1314,6 +1371,100 @@ class WindowPadding {
   }
 }
 
+/// Area of the display that may be obstructed by a hardware feature.This is
+/// populated only on Android.
+///
+/// The [bounds] are measured in logical pixels. On devices with two screens the
+/// coordinate system starts with [0,0] in the top-left corner of the left screen
```

Comment:
May want to clarify orientation of the device, as left/right can also be considered top/bottom